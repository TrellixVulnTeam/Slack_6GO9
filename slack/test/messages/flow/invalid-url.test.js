const { InvalidUrl } = require('../../../lib/messages/flow');

describe('InvalidUrl message rendering', () => {
  test('works', async () => {
    const message = new InvalidUrl('https://amazon.com/atom/atom');
    expect(message.toJSON()).toMatchSnapshot();
  });
});
