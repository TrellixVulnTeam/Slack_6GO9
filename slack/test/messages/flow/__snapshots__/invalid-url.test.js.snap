// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InvalidUrl message rendering works 1`] = `
Object {
  "attachments": Array [
    Object {
      "color": "danger",
      "mrkdwn_in": Array [
        "text",
      ],
      "text": "\`https://amazon.com/atom/atom\` does not appear to be a GitHub link.",
    },
  ],
  "response_type": "ephemeral",
}
`;
